--- lib/vagrant/bundler.rb	2020-04-27 14:19:39.000000000 -0600
+++ lib/vagrant/bundler.rb	2020-04-27 14:20:01.000000000 -0600
@@ -421,8 +421,9 @@
     def vagrant_internal_specs
       # activate any dependencies up front so we can always
       # pin them when resolving
-      Gem::Specification.find { |s| s.name == "vagrant" && s.activated? }.
-        runtime_dependencies.each { |d| gem d.name, *d.requirement.as_list }
+      if (vs = Gem::Specification.find { |s| s.name == "vagrant" && s.activated? })
+        vs.runtime_dependencies.each { |d| gem d.name, *d.requirement.as_list }
+      end
       # discover all the gems we have available
       list = {}
       directories = [Gem::Specification.default_specifications_dir]
